vote/src/vote_account.rs

```diff
@@ -124,6 +126,14 @@ impl VoteAccount {
 
         VoteAccount::try_from(account).unwrap()
     }
+
+    #[cfg(not(all(miri, target_endian = "big")))]
+    pub fn bls_pubkey(&self) -> Option<BLSPubkey> {
+        let bls_pubkey_compressed = self.0.vote_state_view.bls_pubkey_compressed()?;
+        let bls_pubkey_compressed_bytes = BLSPubkeyCompressed(bls_pubkey_compressed);
+        let bls_pubkey = BLSPubkey::try_from(bls_pubkey_compressed_bytes).unwrap();
```

Comment:
Maybe we should just return `BLSPubkeyCompressed` here. This `unwrap()` may or may not succeed, if we somehow have garbage in vote accounts. Also it's probably too inefficient to uncompress every time we call `bls_pubkey()`, we should just uncompress and save the result somewhere in the caller.

Reply:
yeah, good call. `unwrap` here is unsafe. Bad copy+paste from the hacked up PoC